test_ws_client.py

```
@@ -0,0 +1,217 @@
+                print("-" * 60)
+
+                # Виводимо зручний summary
+                if msg_type == "initial_state":
+                    mode_names = {0: "OFF", 1: "ALERT", 2: "WEATHER", 3: "FLAG", 4: "LAMP"}
+                    mode_id = data.get("map_mode_id")
+                    mode_name = mode_names.get(mode_id, f"Unknown({mode_id})")
+
+                    flags16 = data.get("home_alert_flags", 0)
+                    active_alerts = parse_alert_flags(flags16)
+
+                    print(f"✓ Initial state received")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)